laravel-app/app/Http/Controllers/Auth/LoginController.php

```diff
@@ -37,4 +39,13 @@ public function __construct()
     {
         $this->middleware('guest')->except('logout');
     }
+
+    protected function authenticated(Request $request, $user)
```

Comment:
Dont expose your validation request in your controller. Use **Form Request** instead.

Reply:
also you are breaking the principle of Single responsibility in SOLID Principle

Reply:
Changed to a Form Request instead sir. Thanks ❤️

Reply:
Separated the files that does login and verifying the user. Thanks ❤️